scripts/zones/Sacrarium/mobs/Mariselles_Pupil.lua

```diff
@@ -0,0 +1,188 @@
+-----------------------------------
+-- Area: Sacrarium
+--  Mob: Mariselles' Pupils
+-----------------------------------
+local ID = require("scripts/zones/Sacrarium/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/missions")
+-----------------------------------
+local entity = {}
+
+Teleport = function(mob, hideDuration, pos)
```

Comment:
Local functions:

```lua
local function fuctionName(p1, p2, ..., pX)
    -- Insert logic
end
```

Reply:
Cleaned up, moving from former repo syntax to LSB syntax confused me and I forget to insert local. Thanks